gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/internal/ClouddriverService.groovy

```diff
@@ -333,6 +333,15 @@ interface ClouddriverService {
   @PUT('/artifacts/fetch')
   Response getArtifactContent(@Body Map artifact)
 
+  @GET('/artifacts/account/{accountName}/names')
```

Comment:
This doesn't match what gate is currently requesting in https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/src/pipeline/config/triggers/artifacts/ArtifactService.ts#L6 , e.g.
```
/artifacts/helm/account/helm-account/names
```

Reply:
@dbyron0 We have discussed (here: spinnaker/clouddriver#3108) and changed this artifact API a little bit. I prepared pull request https://github.com/spinnaker/deck/pull/6202 according to changes in clouddriver.

Reply:
Great.  Thanks much.